tests/Internal/LuhnTest.php

```diff
@@ -21,10 +21,10 @@ class LuhnTest extends TestCase
     public function testLuhn(string $number) : void
     {
         $rawNumber = substr($number, 0, -1);
-        $checkDigit = substr($number, -1);
+        $checkDigit = (int)substr($number, -1);
```

Comment:
Please add a space after (int). And this can actually be improved as `(int) $number[-1]`!

Reply:
@BenMorel, fixed and improved :).